pkg/web/accounts.go

```diff
@@ -281,7 +282,7 @@ func (s *Server) DeleteAccount(c *gin.Context) {
 	// Content negotiation
 	c.Negotiate(http.StatusOK, gin.Negotiate{
 		Offered:  []string{binding.MIMEJSON, binding.MIMEHTML},
-		HTMLData: gin.H{"AccountID": accountID},
+		HTMLData: scene.Scene{"AccountID": accountID},
```

Comment:
I should've asked when I didn't see `scene.Scene` used in the users PR. I thought it meant we shouldn't update it where `HTMLData` was used. Thank you for making the changes.

Reply:
I don't think scene wasn't merged yet which is why it wasn't in that PR; but we're all caught up now (hopefully).